src/Analyzer.Core.BuiltInRuleTests/TestTemplates/SQLServerAuditingSettings.badtemplate

```diff
@@ -0,0 +1,194 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "storageAccountName": {
+      "type": "string"
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Sql/servers",
+      "name": "ASQLServerName",
+      "apiVersion": "2020-02-02-preview",
+      "properties": {
+        "kind": "v12.0"
+      },
+      "resources": [
+        {
+          "apiVersion": "2020-02-02-preview",
+          "type": "Microsoft.Sql/servers/auditingSettings",
+          "name": "default",
```

Comment:
Regarding the simplification, I tried to do that, only the first resource has all the auditingSettings properties, in case this template can be reused later for another rule, and the rest have only the properties we test on, or is there something else I could safely remove?

Reply:
And about the names, I'm inclined to leave the first resource with a generic name for the template reuse, is that okay? I'm renaming the rest

Reply:
You could probably remove `"apiVersion"` from the resources, but it's not a big reduction, so it's fine either way.